pkg/common/utils/k8s/client.go

```diff
@@ -93,6 +110,12 @@ func ApplyStatefulSet(ctx context.Context, k8sclient client.Client, st *appv1.St
 	return err
 }
 
+func GetStatefulSet(ctx context.Context, k8sclient client.Client, namespace, name string) (*appv1.StatefulSet, error) {
```

Comment:
should add 'apierrors.IsNotFound'

Reply:
this should not clarify, you can read the code from context for judging.